sdk_container/src/third_party/coreos-overlay/sys-libs/libsemanage/libsemanage-3.5.ebuild

```diff
@@ -79,38 +78,18 @@ multilib_src_compile() {
 		LIBDIR="${EPREFIX}/usr/$(get_libdir)" \
 		all
 
-	if multilib_is_native_abi; then
-		building_py() {
-			emake \
-				AR="$(tc-getAR)" \
-				CC="$(tc-getCC)" \
-				LIBDIR="${EPREFIX}/usr/$(get_libdir)" \
-				"$@"
-		}
-		python_foreach_impl building_py swigify
-		python_foreach_impl building_py pywrap
-	fi
+	dotmpfiles "${FILESDIR}/tmpfiles.d/libsemanage.conf"
 }
 
 multilib_src_install() {
 	emake \
 		LIBDIR="${EPREFIX}/usr/$(get_libdir)" \
-		DESTDIR="${ED}" install
-
-	if multilib_is_native_abi; then
-		installation_py() {
-			emake DESTDIR="${ED}" \
-				LIBDIR="${EPREFIX}/usr/$(get_libdir)" \
-				install-pywrap
-			python_optimize # bug 531638
-		}
-		python_foreach_impl installation_py
-	fi
-}
+		SHLIBDIR="/usr/$(get_libdir)" \
+		DEFAULT_SEMANAGE_CONF_LOCATION="/usr/lib/selinux/semanage.conf" \
+		DESTDIR="${ED}" \
+		install
 
-multiib_src_install_all() {
-	python_setup
-	python_fix_shebang "${ED}"/usr/libexec/selinux/semanage_migrate_store
+	dotmpfiles "${FILESDIR}/tmpfiles.d/libsemanage.conf"
```

Comment:
I think we don't need this any more - I think we can just install semanage.conf in `/etc`. But maybe we will need to install a compatibility symlink (`/usr/lib/selinux/semanage.conf` -> `/usr/share/flatcar/etc/selinux/semanage.conf`) with coreos-base/misc-files…